packages/blazor-workspace/Tests/NimbleBlazor.Tests.Acceptance.Client/Pages.InteractiveWebAssembly/DialogOpenAndClose.razor

```diff
@@ -0,0 +1,30 @@
+﻿@page "/InteractiveWebAssembly/DialogOpenAndClose"
+@namespace NimbleBlazor.Tests.Acceptance.Client.Pages
+@inherits LayoutComponentBase
+@rendermode RenderMode.InteractiveWebAssembly
```

Comment:
Currently this test is an exact copy of the pre-existing DialogAndOpenClose test, except using `InteractiveWebAssembly` render mode instead of `InteractiveServer`.

This gets us some test coverage for the Wasm render mode. My initial aim was to run all of the existing tests under both `InteractiveServer` and `InteractiveWebAssembly` render modes. I wasn't able to get that all working in a reasonable amount of time, but pretty sure it should be possible.